GBA/bg.zig

```diff
@@ -93,6 +121,19 @@ pub const AffineScreenEntry = u8;
 pub const TextScreenBlock = [1024]TextScreenEntry;
 pub const screen_block_ram: [*]volatile TextScreenBlock = @ptrCast(display.vram);
 
+pub fn screenBlockMap(block: u5) [*]volatile bg.TextScreenEntry {
+    return @ptrCast(&screen_block_ram[block]);
+}
+
+/// Copy memory into a screenblock, containing background layer data.
+/// Note that screenblocks and charblocks share the same VRAM.
+pub fn memcpyScreenBlock(block: u5, data: []const u8) void {
+    gba.mem.memcpy32(
```

Comment:
This is copied from the mode4flip example. I discovered the hard way that this actually breaks if the data is incidentally not aligned on a 16-bit word boundary. (Causing memcpyGeneric to be used instead of memcpyAligned.)